client/my-sites/site-settings/settings-reading/main.tsx

```diff
@@ -70,12 +79,12 @@ type Fields = {
 	posts_per_page?: number;
```

Comment:
Oh yes! 🙈  It showcases that something is actually not properly typed because we did not get a type error there. Never-the-less we should add it there. The types also act as a form of documentation. It is good to know what values are we expected to see in a given object. 👍

Reply:
I just fixed this.